WordPress/src/main/java/org/wordpress/android/ui/reader/subfilter/SubfilterListItemMapper.kt

```diff
@@ -0,0 +1,92 @@
+package org.wordpress.android.ui.reader.subfilter
+
+import com.google.gson.Gson
+import org.wordpress.android.datasets.ReaderBlogTable
+import org.wordpress.android.models.ReaderBlog
+import org.wordpress.android.models.ReaderTagType
+import org.wordpress.android.ui.reader.subfilter.SubfilterListItem.ItemType.SITE
+import org.wordpress.android.ui.reader.subfilter.SubfilterListItem.ItemType.SITE_ALL
+import org.wordpress.android.ui.reader.subfilter.SubfilterListItem.ItemType.TAG
+import org.wordpress.android.ui.reader.subfilter.SubfilterListItem.Site
+import org.wordpress.android.ui.reader.subfilter.SubfilterListItem.SiteAll
+import org.wordpress.android.ui.reader.subfilter.SubfilterListItem.Tag
+import org.wordpress.android.ui.reader.utils.ReaderUtils
+import javax.inject.Inject
+
+private data class MappedItem(
+    val type: Int,
+    var feedId: Long = 0,
+    var blogId: Long = 0,
+    var tagSlug: String = "",
+    var tagType: Int = 0
+)
+
+class SubfilterListItemMapper @Inject constructor() {
```

Comment:
I'd add a few unit tests for this class.

Reply:
Agree 👌. Added here 4e937721a5282bab82594ce00be957cbdd2fe88f